src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -38,21 +38,93 @@ class OrganizationDeveloperSettings extends AsyncView {
     publishRequestSentryApp(this.api, app);
   };
 
-  renderBody() {
+  renderApplicationRow = app => {
     const {organization} = this.props;
+    return (
+      <SentryApplicationRow
+        key={app.uuid}
+        app={app}
+        organization={organization}
+        onRemoveApp={this.removeApp}
+        onPublishRequest={this.publishRequest}
+        showInstallationStatus={false}
+      />
+    );
+  };
+
+  renderInternalIntegrations() {
+    const {orgId} = this.props.params;
+    const integrations = this.state.applications.filter(app => app.status === 'internal');
+    const isEmpty = integrations.length === 0;
+
+    const action = (
+      <Button
+        priority="primary"
+        size="small"
+        to={`/settings/${orgId}/developer-settings/new-internal/`}
+        icon="icon-circle-add"
+      >
+        {t('New Internal Integration')}
+      </Button>
+    );
+
+    return (
+      <Panel>
+        <PanelHeader hasButtons={true}>
+          {t('Internal Integrations')}
+          {action}
+        </PanelHeader>
+        <PanelBody>
+          {!isEmpty ? (
+            integrations.map(this.renderApplicationRow)
+          ) : (
+            <EmptyMessage>
+              {t('No internal integrations have been created yet.')}
+            </EmptyMessage>
+          )}
+        </PanelBody>
+      </Panel>
+    );
+  }
+
+  renderExernalIntegrations() {
     const {orgId} = this.props.params;
+    const integrations = this.state.applications.filter(app => app.status !== 'internal');
+    const isEmpty = integrations.length === 0;
+
     const action = (
       <Button
         priority="primary"
         size="small"
-        to={`/settings/${orgId}/developer-settings/new/`}
+        to={`/settings/${orgId}/developer-settings/new-external/`}
         icon="icon-circle-add"
       >
-        {t('Create New Integration')}
+        {t('New External Integration')}
       </Button>
     );
 
-    const isEmpty = this.state.applications.length === 0;
+    return (
+      <Panel>
+        <PanelHeader hasButtons={true}>
+          {t('External Integrations')}
```

Comment:
Will users grok what 'external' means in this context? Is it for external services?  Would calling them 'public integrations' be easier to understand?

Reply:
@markstory Good question. Let me ask @mnoble on our channel.

Reply:
@markstory making the change to `public`